cluster/saltbase/salt/kube-apiserver/kube-apiserver.manifest

```diff
@@ -43,10 +43,12 @@
 
 {% set cert_file = "--tls_cert_file=/srv/kubernetes/server.cert" -%}
 {% set key_file = "--tls_private_key_file=/srv/kubernetes/server.key" -%}
+{% set client_ca_file = "--client_ca_file=/dev/null" -%}
```

Comment:
This broke non-GCE providers I think.  Right now, the kube-apiserver outputs errors like the following:

```shell
F0427 21:05:35.065323       1 server.go:247] Invalid Authentication Config: error reading /dev/null: could not read any certificates
```

Is there any reason why we could not default the value to the empty string?

```
{% set client_ca_file = "" -%}
```